### ISSUE TITLE
RPM files

### ISSUE THREAD

**Issue by loikki** [jbeder/yaml-cpp#1132]
Hi,

Are you interested in releasing RPM files (and maybe DEB)? I have implemented everything on gitlab and can push it here if needed.

https://gitlab.com/loikki/yaml-cpp-rpm

Best regards,

**Comment by jbeder** [comment]
Is an RPM file a compiled object or source? If it's compiled, then I don't think so, but if it's source, then you can put it in contrib/

**Comment by loikki** [comment]
RPM is the packaging system of Fedora. It is compiled, so the idea is not to put it inside the git repository but to add it to the CI/CD pipeline in order to provide it during the releases.

**Comment by loikki** [comment]
Yes but they are not necessary very reactive. For example Fedora does not have the version 0.7.0 yet (which is why I created my repo with the CI/CD pipeline to generate the RPM file)

**Comment by jbeder** [comment]
Oh, I think you mean that when I do a release, it creates a binary object that you can download from github?

**Comment by loikki** [comment]
Exactly :)

**Comment by jbeder** [comment]
I see. If it's just part of the CI, then sure, feel free to submit a PR.
